frontend/components/product-list/sections/FilterableProductsSection/FacetsAccordion.tsx

```diff
@@ -25,8 +25,10 @@ type FacetsAccordianProps = {
 };
 
 const initialExpandedFacets = ['facet_tags.Item Type'];
+let isemptyFacetState = 0;
```

Comment:
I don't think it's a good idea to keep state in a variable outside the component like this.

Reply:
I will try to find an alternative. Thanks!